lib/chef/knife/bmcs_server_show.rb

```diff
@@ -32,6 +85,7 @@ def run
           next unless vnic.lifecycle_state == 'ATTACHED'
           begin
             vnic_info = network_client.get_vnic(vnic.vnic_id, {})
+            add_vnic_details(vnic_info.data)
```

Comment:
I realize that this might change with multi-vnic support, but is it worth doing this for now for secondary vnics?

Reply:
The patches for the multi-vnic support do this lookup already.